statsmodels/tsa/statespace/initialization.py

```diff
@@ -602,11 +602,12 @@ def __call__(self, index=None, model=None, initial_state_mean=None,
         # the blocks and return those
         else:
             for block_index, init in self.blocks.items():
-                out = init(
-                    index=tuple(np.array(index)[block_index, ]),
-                    model=model, initial_state_mean=initial_state_mean,
-                    initial_diffuse_state_cov=initial_diffuse_state_cov,
-                    initial_stationary_state_cov=initial_stationary_state_cov)
+                # TODO: Should something be done with the object returned
+                # by init?
```

Comment:
@ChadFulton last thing.

Reply:
Ah, no, I think they're set in place at each recursion. I guess maybe I was originally planning something different. If you don't mind, can you replace:

```python
# Otherwise, if using blocks, combine the elements returned by each of
# the blocks and return those
```

with:

```python
# Otherwise, if using blocks, recursively initialize them (values will be set in-place)
```